src/main/java/io/mosip/print/dto/BaseRestResponseDTO.java

```diff
@@ -0,0 +1,25 @@
+package io.mosip.print.dto;
+
+import java.io.Serializable;
+
+import lombok.Data;
+
+/**
+ * Instantiates a new base request response DTO.
+ * @author Rishabh Keshari
+ */
+@Data
+public class BaseRestResponseDTO implements Serializable {
+	
+	private static final long serialVersionUID = 4246582347420843195L;

```

Comment:
Should we start `serialVersionUID` with value `1L` like other class make it consistent between versions?